docs/Home.md

```diff
@@ -1,3 +1,13 @@
-Welcome to the android-templates wiki! 
+Welcome to the android-templates wiki!
 
-More coming soon...
\ No newline at end of file
+The purpose of our android-templates repository is to quickly start a new project with all the **must-have** requirements. Other **nice-to-have** features need to be considered as an optional setting when starting a new project.
+
+Planning to add a new feature? Ask yourself the following questions to define if it's a **must-have** requirement:
+
+- Is the feature frequently used in other projects? (Ex: Glide, Retrofit, ...)
+- Does the feature increase productivity of the team? (Ex: Firebase, Proxyman, ...)
```

Comment:
Firebase is huge, and if it's about Analysis we don't always have it.
Maybe you are mentioning Firebase Distribution or Crashlytics? Let's be explicit